app/views/layouts/application.haml

```diff
@@ -71,7 +71,7 @@
                 %span &nbsp;|&nbsp;
                 = link_to "Sign up", new_user_path
             = link_to root_path do
-              = image_tag "layouts/logo.png", alt: 'Glowfic Constellation home'
+              = image_tag "layouts/logo.png", alt: 'Glowfic Constellation home', height: '20px'
```

Comment:
It looks like this is 30px when I load up the site itself – was this a typo? Also, when I try this myself, even with `30`, it seems to change the size of the logo unless we change the `padding` CSS declaration to `margin`.

(Aside: MDN says this should be declared without `px`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#attr-height)

Reply:
Ooh, it's rendered as 20px on mobile due to the media query. Even so, this attribute seems like it's meant to be based on the intrinsic size of the image, which seems to be 30px tall? I think that and the padding -> margin CSS change, or putting the padding on the link instead of the image, would solve it, but we might need to check how it renders in practice to be sure.